_includes/header.html

```diff
@@ -106,7 +106,4 @@
             </div>
         </div>
     </nav>
-</div>
-
-<!-- make sure this get included on every page -->
-<!--[if !IE]><script>fixScale(document);</script><![endif]-->
+</div>
```

Comment:
newline at end of file. Since this is an include, it could have bad effects if it is missing.